java-checks/src/test/java/org/sonar/java/checks/naming/KeywordAsIdentifierCheckTest.java

```diff
@@ -25,8 +25,13 @@
 public class KeywordAsIdentifierCheckTest {
 
   @Test
-  public void test() {
-    JavaCheckVerifier.verify("src/test/files/checks/naming/KeywordAsIdentifierCheck.java", new KeywordAsIdentifierCheck());
+  public void test_enum() {
+    JavaCheckVerifier.verify("src/test/files/checks/naming/KeywordAsIdentifierCheck_enum.java", new KeywordAsIdentifierCheck(), 4);
+  }
+
+  @Test
+  public void test_underscore() {
+    JavaCheckVerifier.verify("src/test/files/checks/naming/KeywordAsIdentifierCheck_underscore.java", new KeywordAsIdentifierCheck(), 8);
```

Comment:
This file doesn't use anything that requires Java greater than 4, thus actually no need to split :wink: Sorry that I did not notice this before.